test/e2e/petset.go

```diff
@@ -67,7 +67,7 @@ const (
 // Time: 25m, slow by design.
```

Comment:
I checked the logs on GCE, all stateful set tests (other than disruptive and clustered app ones) took <200s. In this case I'll remove `Slow` tag on them.